### ISSUE TITLE
dxt5 is broken

### ISSUE THREAD

**Issue by klaxons1** [smb123w64gb/vtfTrans-rs#3]
![image](https://github.com/user-attachments/assets/fde305ec-cd0d-4573-a62b-6ae1ffa31dbe)

**Comment by smb123w64gb** [comment]
Yes... Ive got allot to get this fully operational. I can see what improvements i can do to get It into a more stable manner. For the time being, is this from HL2X XTF to VTF or converted VTF to XTF to VTF.. Somewhere on my conversion there is some issues there as well.

**Comment by smb123w64gb** [comment]
OK on first look, i must of messed up my Preload header to lowres image.
XTF is formed as such,

Main Texture image - in this case 256 x 256 UV88
Low Res image - Similar to VTF but shares the same format as before(The same texture file from PC is UV88+DXT1)
Preload image - This is loaded into ram as a failsafe for xbox to render with before the full sized image is loaded from hdd.

So in short, I've been just abusing the Preload as my Low Res since it makes since most of the time.... However i need to prob add a texture resizing bit of code to create them since they don't follow the same Low res texture that VTF are used too. 
That and UV88 may not be swizzled or vice versa.
So ill do some work to fix my VTFedit viewing, and so on.

Also Xbox textures are almost the same as PC just 1/2 in resolution.
